tests/test_termset_pairwise_similarity.rs

```diff
@@ -132,3 +131,62 @@ fn test_large_termset_pairwise_similarity() {
     let result = rss.termset_pairwise_similarity(&entity1, &entity2);
     dbg!(&result);
 }
+
+#[test]
+#[ignore]
+#[cfg_attr(feature = "ci", ignore)]
+fn test_ancestor_label_presence() {
+    let mut db_path = PathBuf::new();
+    if let Some(home) = std::env::var_os("HOME") {
+        db_path.push(home);
+        db_path.push(".data/oaklib/phenio.db");
+    } else {
+        panic!("Failed to get home directory");
+    }
+    let db = Some(db_path.to_str().expect("Failed to convert path to string"));
+    let predicates: Option<Vec<Predicate>> = Some(vec![
+        "rdfs:subClassOf".to_string(),
+        "BFO:0000050".to_string(),
+        "UPHENO:0000001".to_string(),
+    ]);
+
+    let mut rss = RustSemsimian::new(None, predicates, None, db);
+    rss.update_closure_and_ic_map();
+
+    let entity1: HashSet<TermID> = HashSet::from([
+        "MP:0010771".to_string(),
+        "MP:0002169".to_string(),
+        "MP:0005391".to_string(),
+        "MP:0005389".to_string(),
+        "MP:0005367".to_string(),
+    ]);
+    let entity2: HashSet<TermID> = HashSet::from([
+        "HP:0004325".to_string(),
+        "HP:0000093".to_string(),
+        "MP:0006144".to_string(),
+    ]);
+    let result = rss.termset_pairwise_similarity(&entity1, &entity2);
+
+    dbg!(&result);
```

Comment:
testing for not None is good, but shouldn't we be testing that the label is correct when it is not None?

Reply:
(possibly we are testing this somewhere else but I can't find it)

Reply:
The `for` loop below does the check for every `ancestor_label`

Reply:
oh you mean the correct label. Good point!

Reply:
https://github.com/monarch-initiative/semsimian/blob/cbb504cf420187d8f87a54e75ec023d439b02859/src/db_query.rs#L382-L388